Source/WebKit/UIProcess/ios/ViewGestureControllerIOS.mm

```diff
@@ -88,6 +88,13 @@ - (instancetype)initWithViewGestureController:(WebKit::ViewGestureController*)ge
 
 - (void)invalidate
 {
+    if (_gestureRecognizerView) {
+        if (RetainPtr backRecognizer = [_backTransitionController gestureRecognizer])
+            [_gestureRecognizerView.get() removeGestureRecognizer:backRecognizer.get()];
+        if (RetainPtr forwardRecognizer = [_forwardTransitionController gestureRecognizer])
+            [_gestureRecognizerView.get() removeGestureRecognizer:forwardRecognizer.get()];
```

Comment:
It looks like this comment was missed: https://github.com/WebKit/WebKit/pull/55345#discussion_r2624525707

There's no need for `.get()` on `_gestureRecognizerView`. A method call on `RetainPtr` will work on the underlying pointer.

Not urgent, but could clean in a follow-up PR.